tutorials/packaging/create-your-first-snap.md

```diff
@@ -659,24 +678,29 @@ beta       ^          ^
 edge       ^          ^
 ```
 
-You should receive an email telling you that your snap is pending review (automatic checking). If you are not using any reserved interfaces and security checks are passing, it would be available to the user (in the **candidate**, **beta** and **edge** channels in this case) to your users via a simple:
-
+You should receive an email telling you that your snap is pending review (automatic checking). If you are not using any reserved interfaces and security checks are passing, it would be available to the user (in the **candidate** channel in this case) to your users via a simple:
 
 ```bash
-$ sudo snap install hello-didrocks --channel=candidate
+sudo snap install hello-didrocks --channel=candidate
 ```
 
-
 negative
-: As you uploaded an amd64 binary, only people on amd64 machine will get access to this snap. You can either only focus on one architecture to support, build yourself for all supported architectures your snap, or use **launchpad snap build service** to push your `snapcraft.yaml`, and get resulting snaps built on all architectures for you! More information are [available here].
+: As you uploaded an amd64 binary, only people on amd64 machine will get access to this snap. You can either only focus on one architecture to support, build yourself for all supported architectures your snap, or use [build.snapcraft.io](https://build.snapcraft.io) to push your `snapcraft.yaml`, and get resulting snaps built on all architectures for you!
```

Comment:
the second use of `amd64` makes this sound AMD-specific to me, rather than simply a 64-bit build for x86 systems. Also `build yourself for all...` could probably be something like `build a binary for each architecture you wish to support`.